src/albums/dto/update-album.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+  Min,
+} from 'class-validator';
+import { CreateAlbumDto } from './create-album.dto';
+
+export class UpdateAlbumDto extends PartialType(CreateAlbumDto) {
```

Comment:
Completely duplicates CreateAlbumDto, I believe it's created by Nest.js generator for PATCH method in the 1st place.

Reply:
Yep, maybe it could be better to use only AlbumDto. I just followed flow of the "nest g resource"